src/bewit.rs

```diff
@@ -35,10 +35,11 @@ impl<'a> Bewit<'a> {
 
     /// Generate the fully-encoded string for this Bewit
     pub fn to_str(&self) -> String {
+        use base64::display::Base64Display;
```

Comment:
why embed this in the function, rather than toplevel?

Reply:
Nothing else really needed it. If you'd prefer I move it I can.

Reply:
Just trying to get a sense for proper style.  In other languages the equivalents (import, require, etc.) go at the top unless there's some special need to delay the load or scope the symbols.  If this seems normal to you then I'm happy with it :)